docs/data/pages.ts

```diff
@@ -15,7 +15,19 @@ const pages = [
       { pathname: '/x/react-data-grid/getting-started' },
       { pathname: '/x/react-data-grid/migration-v4', title: 'Migration from v4' },
       { pathname: '/x/react-data-grid/layout' },
-      { pathname: '/x/react-data-grid/columns' },
+      {
+        pathname: '/x/react-data-grid/columns',
+        children: [
+          { pathname: '/x/react-data-grid/column-definition' },
```

Comment:
Did you merge your PR after the monorepo has been updated? It's showing the menu items as "Column Definition", not "Column definition".

Reply:
The monorepo is update to date yes